9c-main/chart/templates/validator.yaml

```diff
@@ -78,6 +78,18 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: IpRateLimiting__EnableEndpointRateLimiting
+            value: "true"
+          - name: IpRateLimiting__GeneralRules__2__Endpoint
+            value: "*:/graphql"
+          - name: IpRateLimiting__GeneralRules__2__Period
+            value: 1s
+          - name: IpRateLimiting__GeneralRules__2__Limit
+            value: "1"
+          - name: IpRateLimiting__IpBanMinute
+            value: "1"
+          - name: IpRateLimiting__IpWhiteList__1
+            value: "157.245.68.243"
```

Comment:
```suggestion
```
You can remove this part becaues it's the DappRadar service IP that uses the full-state node 🙏